tools/accuracy_checker/accuracy_checker/annotation_converters/antispoofing.py

```diff
@@ -0,0 +1,110 @@
+                    description='path to json file with dataset meta (e.g. label_map)', optional=True
+                ),
+                'annotations': PathField(
+                    description='path to json file with dataset annotations ({index : {path: ..., labels: ..., boxes: ... (optional)}})', optional=False
+                )
+            }
+        )
+        return configuration_parameters
+
+    def configure(self):
+        super().configure()
+        self.data_dir = self.get_value_from_config('data_dir')
+        self.annotations = self.get_value_from_config('annotations')
+        self.meta = self.get_value_from_config('meta_file')
+
+    def convert(self, check_content=False, progress_callback=None, progress_interval=100, **kwargs):
+        """Reads data from disk and returns dataset in converted for AC format
+
+        Args:
+            check_content (bool, optional): Check if content is valid. Defaults to False.
+            progress_callback (bool, optional): Display progress. Defaults to None.
+            progress_interval (int, optional): Units to display progress. Defaults to 100 (percent).
+
+        Returns:
+            [type]: Converted dataset
+        """
+        annotation_tuple = self.generate_annotations()
+        annotations = []
+        content_errors = None if not check_content else []
+        meta = self.generate_meta()
+        labels_to_id = meta['label_map']
+        num_iterations = len(annotations)
+
+        for i, (img_name, label, bbox) in enumerate(annotation_tuple):
+            image_annotation = ClassificationAnnotation(img_name, label)
+            if bbox:
+                image_annotation.metadata['rect'] = bbox
+            annotations.append(image_annotation)
+
+            if check_content:
+                if not check_file_existence(os.path.join(self.data_dir, img_name)):
+                    content_errors.append(f'{img_name}: does not exist')
+            if progress_callback is not None and i % progress_interval == 0:
+                    progress_callback(i / num_iterations * 100)
+
+        return ConverterReturn(annotations, meta, content_errors)
+
+    def generate_meta(self):
+        if not self.meta:
+            return {'label_map': {'real': 0, 'spoof': 1}}
+        dataset_meta = read_json(self.meta)
+        label_map = dataset_meta.get('label_map')
+        dataset_meta['label_map'] = label_map or {'real': 0, 'spoof': 1}
+        return dataset_meta
+
+    def generate_annotations(self):
+        ''' read json file with images paths and return
+        list of the items'''
+        annotation_store = []
+        dataset_annotations = read_json(self.annotations)
+        for index in dataset_annotations:
+            path = dataset_annotations[index]['path']
+            # this branch will be needed if multi-task labels given to input
+            if len(dataset_annotations[index]['labels']) > 1:
+                target_label = dataset_annotations[index]['labels'][43]
```

Comment:
what is the magic number 43? :)

Reply:
There are 43 labels for this dataset. Each label represents a specific task. These labels had been using for the multi-task training approach. The final model predicts only the main task, so this is 43 label in JSON annotation.